ccdproc/core.py

```diff
@@ -580,13 +580,16 @@ def sigma_func(arr):
     ----------
     arr : `~ccdproc.CCDData` or `~numpy.ndarray`
         Array whose deviation is to be calculated.
+        
+    axis : None or int or tuple of ints, optional
+        Axis or axes along which a sum is performed. The default (axis = None) is perform a sum over all the dimensions of the input array. axis may be negative, in which case it counts from the last to the first axis.
```

Comment:
Could you split that line into multiple ones it's recommended that lines are not too much over 80 letters per row?  Also I find it a bit misleading: It's not a sum that is performed it's a "sigma function". The word "axis" should be capitalized when it's at the beginning of the sentence.

I would suggest:

```
Axis or axes along which the function is performed.
If ``None`` (the default) it is performed over all the dimensions of the input array.
The axis argument can also be negative, in which case it counts from the last to the first axis.
```